hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileRange.java

```diff
@@ -52,4 +54,8 @@ public interface FileRange {
    * @param data the future of the ByteBuffer that will have the data
    */
   void setData(CompletableFuture<ByteBuffer> data);
+
+  static FileRange createFileRange(long offset, int length) {
```

Comment:
1. javadoc
2. where tests use the contructor themselves, they should switch to this method